src/utils.rs

```diff
@@ -0,0 +1,48 @@
+use once_cell::sync::Lazy;
+
+use crate::errors::ApiError;
+
+
+pub static SECRET_KEY: Lazy<String> =
+    Lazy::new(|| std::env::var("SECRET_KEY").unwrap_or_else(|_| "0123".repeat(16)));
+
+const _SALT: &[u8] = b"supersecuresalt";
```

Comment:
For the underscored variable here and the functions below, either remove or add comments as to why they ignored.

Reply:
Repair later